functions.php

```diff
@@ -154,6 +154,11 @@ function c4aa_setup_theme_supported_features() {
 			'slug' => 'caa-grey',
 			'color' => '#968c83',
 		),
+		array(
+			'name' => __( 'C4AA Black', 'themeLangDomain' ),
```

Comment:
@slambert still need to update these! should be:
`'name' => __( 'C4AA Black', 'c4aa' ),`

Reply:
This should be done and in commit 9ea31b1

https://github.com/thec4aa/c4aa-org/commit/9ea31b1bb2f7aa46eed72bed43469fb506d0de4d

??

Reply:
Oh - I guess what the PR is showing is outdated? Meh, merging!